lib/consoletest.pm

```diff
@@ -8,7 +8,7 @@ use strict;
 use warnings;
 use testapi;
 use known_bugs;
-use version_utils 'is_public_cloud';
+use version_utils qw(is_public_cloud is_openstack);
```

Comment:
leftover?

Reply:
Nope, I have just forgotten to add `return` right after we come back from the super post_fail_hook. Not sure whether the log console part is needed in case of PC/OpenStack jobs. Wdyt ?

Reply:
I don't think it's needed, everything is happening remotely.. Local logs won't help much as we are booting the PC tools image which is not the SUT.